recipes/systemc_accellera/2.3.3/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SystemcAccelleraConan(ConanFile):
+    name = "systemc_accellera"
```

Comment:
Is `systemc_accellera` actually the better name instead of just `systemc`?

Vcpkg used `systemc` https://github.com/Microsoft/vcpkg/tree/master/ports/systemc

Reply:
I just read @madebr reasoning.
 
Not sure about this one..

Reply:
I was just thinking loudly and did not check vcpkg's name.
`systemc` is fine to me too.
Other EDA's should have open sourced their systemc implementation 😃

Reply:
@vanwinkeljan Sorry for the overhead

Reply:
renamed